### PR TITLE
feat: add config flag 'force-bump-patch-version'

### PR DESCRIPTION
**Pull request by timoknapp**
Hi @christophwitzko ,

Based on #98 as promised the related PR. In case you want to rephrase the flag, just let me know.

Cheers
Timo

### CODE REVIEW

File: pkg/semrel/semrel_test.go [hunk 1]
```diff
@@ -39,28 +39,30 @@ func TestApplyChange(t *testing.T) {
...
+		{"0.0.0", MajorChange, "1.0.0", true, false},
 
-		{"1.0.0", NoChange, "", false},
-		{"1.0.0", PatchChange, "1.0.1", false},
-		{"1.0.0", MinorChange, "1.1.0", false},
-		{"1.0.0", MajorChange, "2.0.0", false},
-		{"0.1.0", NoChange, "1.0.0", false},
-		{"0.1.0", NoChange, "", true},
+		{"1.0.0", NoChange, "", false, false},
+		{"1.0.0", PatchChange, "1.0.1", false, true},
```
Comment (christophwitzko) [inline pkg/semrel/semrel_test.go:56]: Here should be passed `NoChange`

**Comment by christophwitzko** [comment]
LGTM 👍

**Comment by christophwitzko** [comment]
Hey Timo, thanks for the PR. ✨

File: pkg/semrel/semrel_test.go (hunk 1 above)
Comment (timoknapp) [reply pkg/semrel/semrel_test.go]: Okay understood, I just adapted the line in the PR

**Review by christophwitzko — APPROVED** [review]

**Comment by timoknapp** [comment]
Hi @christophwitzko ,

thanks for merging my PR! Do you already know when you are going to release the next version including my change?

Cheers
Timo

**Comment by christophwitzko** [comment]
@timoknapp v2.13.0 is now the latest release.

**Comment by christophwitzko** [comment]
You are welcome. Thank you again for the PR. ☺️

**Pull request merged**